Microsoft.HealthVault.Fhir/Transformers/ObservationToBloodGlucose.cs

```diff
@@ -20,7 +20,11 @@ internal static BloodGlucose ToBloodGlucose(this Observation observation)
             var bloodGlucose = observation.ToThingBase<BloodGlucose>();
 
             bloodGlucose.Value = ObservationToHealthVault.GetThingValueFromQuantity<BloodGlucoseMeasurement>(observation.Value as Quantity);
-            bloodGlucose.When = ObservationToHealthVault.GetHealthVaultTimeFromEffectiveDate(observation.Effective);
+
+            if (observation.Effective == null)
```

Comment:
Close/open brackets. Also, if we are throwing an exception a more descriptive exception is better: "Effective field with a valid Effectiva date on observation is required". Although I am not sure why we should throw an exception here, what made you modify this?